src/core/scene/a-scene.js

```diff
@@ -317,10 +395,33 @@ var AScene = module.exports = registerElement('a-scene', {
 
         if (this.isPlaying) {
           this.tick(time, timeDelta);
+          this.tock(time, timeDelta);
+        }
+
+        if (this.behaviors.postRenderTick.length > 0 || this.behaviors.postRenderTock.length > 0) {
+          // Post processing setup has been modified, emit a notification.
+          if (this.postNeedsNotify) {
+            var bs = this.behaviors;
+            this.postNeedsNotify = false;
+            this.emit('post-modified', { tick: bs.postRenderTick, tock: bs.postRenderTock });
+          }
+
+          this.renderer.render(this.object3D, camera, this.renderTarget);
+
+          this.time = time;
```

Comment:
it won't even run if there's not post processing but what kind of debug should we output? I don't like the idea of outputing on the console every frame. Three.js has a stats object inside where it collects various metrics like draw calls etc Maybe we should do something similar?